backend/handlers.go

```diff
@@ -1,16 +1,16 @@
 package backend
 
 import (
+	"bytes"
 	"context"
 	"encoding/json"
-	"io"
-	"net/http"
-	"os"
-	"strings"
+	"fmt"
 	"github.com/honeynet/ochi/backend/entities"
-
 	"github.com/julienschmidt/httprouter"
 	"google.golang.org/api/idtoken"
+	"io"
+	"net/http"
+	"os"
```

Comment:
The build-in imports should be at the top. This is an indication that you are not using `gofmt`